moveit_core/robot_model/include/moveit/robot_model/log.h

```diff
@@ -0,0 +1,54 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2019, Acutronic Robotics AG
+ *  All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above
+ *     copyright notice, this list of conditions and the following
+ *     disclaimer in the documentation and/or other materials provided
+ *     with the distribution.
+ *   * Neither the name of Willow Garage nor the names of its
+ *     contributors may be used to endorse or promote products derived
+ *     from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ *  FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ *  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ *  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ *  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+ *********************************************************************/
+
+/* Author: Víctor Mayoral Vilches */
+
+#ifndef MOVEIT_CORE_ROBOT_MODEL_LOG_
+#define MOVEIT_CORE_ROBOT_MODEL_LOG_
+
+#include "rclcpp/rclcpp.hpp"
+
+namespace moveit
+{
+namespace core
+{
+
+// Logger
+rclcpp::Logger logger_robot_model = rclcpp::get_logger("robot_model");
```

Comment:
Does this need to be in it's own file or can it live in robot_model.h?

Reply:
+1 to keeping it at the top of every file, such as in this [example](https://github.com/ros-planning/moveit2/pull/33/files#diff-15d4dc0e9e1eee143f7576803dfe8ea2R57)

Reply:
> Does this need to be in it's own file or can it live in robot_model.h?

@mlautman I initially followed this approach to avoid having LOGGERs across different source files (which IMHO at the time of writing, it was harder to maintain). Frankly and after having tinkered with many modules over the last weeks, I'm now much more convinced about @davetcoleman's argument so I followed this path at 80e49b00936e36a4640bb9303a02f775cfeacf92.

Reply:
Sounds good.